main.go

```diff
@@ -175,9 +176,56 @@ func (p DockerAuthZPlugin) evaluate(ctx context.Context, r authorization.Request
 	return p.evaluatePolicyFile(ctx, r)
 }
 
+type BindMount struct {
+  Source string
+  ReadOnly bool
+  Resolved string
+}
+
+func listBindMounts(body map[string]interface{}) ([]BindMount) {
+	var result []BindMount
+	
+	hostConfig, ok := body["HostConfig"].(map[string]interface{})
+	if ok {
+                binds, ok := hostConfig["Binds"].([]interface{})
+                if ok {
+			for _, v := range binds {
```

Comment:
Indentation looks off here. Could you please gofmt the files included here? Or is this some GitHub display issue?

Reply:
Fixed - it was my fault, I wasn't doing this in an environment where gofmt was on by default.

Reply:
We should probably include a check for that as part of the build steps, but only so much time :)